### ISSUE TITLE
Add explanation of difference to other templates

### ISSUE THREAD

**Issue by koppor** [latextemplates/scientific-thesis-template#25]
<s>@johannesbottcher [recommended](https://twitter.com/templateGundar/status/686207703618838528) to have a look at [thesis.cls](https://www.ctan.org/tex-archive/macros/latex/contrib/thesis) and to include important snippets.</s>

There are many other templates around there is a [confusion about templates](https://github.com/johannesbottcher/templateConfusion). Thus, we should provide what other templates we are aware of and clearly state the differences to help readers to decide. A good example for that is the documentation of the [listings](https://www.ctan.org/pkg/listings) package, which states the difference to most competitors.

- [ ] https://github.com/andygrunwald/FOM-LaTeX-Template
- [ ] Hagenberg: https://github.com/Digital-Media/HagenbergThesis - very extensive thesis template
- [ ] Template by @pospiech https://github.com/pospiech/latex/tree/master/latexthesistemplate/template / http://www.matthiaspospiech.de/latex/vorlagen/allgemein/
- [ ] [cleanthesis](https://github.com/derric/cleanthesis/)
- [ ] [classicthesis](https://bitbucket.org/amiede/classicthesis/wiki/Home) - beware; this is GPL-licensed: You have to publish the whole LaTeX source of your thesis. Siehe auch https://bitbucket.org/amiede/classicthesis/issues/123/gpl-suitable
- [ ] [basis](https://github.com/ekleinod/basis)
- [ ] http://www.ctan.org/pkg/cquthesis
- [ ] https://ctan.org/pkg/ku-template
- [ ] https://ctan.org/pkg/brandeis-dissertation
- [ ] https://ctan.org/pkg/hithesis
- [ ] Templates listed at https://ctan.org/topic/dissertation and https://ctan.org/topic/thesis-template
- [ ]  https://github.com/iswunistuttgart/latex-templates
- [ ] https://github.com/MrDongdongLin/xtuthesis - also has minimal examples. With explanations in Chinese.
- [ ] Templates listed at https://tex.stackexchange.com/questions/326/latex-templates-for-writing-a-thesis
- [ ] Markdown-based templates such as https://github.com/tompollard/phd_thesis_markdown.

Maybe add them to https://github.com/latextemplates/latextemplates.github.io/blob/master/index.md#templates-by-others?

## Lists of theses

- http://uweziegenhagen.de/?p=1005

**Comment by johannesbottcher** [comment]
No, not at all. I was referring to the Thesis template from Sunil Patel and i was ironic. [See why](https://github.com/johannesbottcher/templateConfusion/blob/master/ourFamousThesisTemplate.md). 

The class linked on CTAN is 20 years! old, much has been done in the meantime.

**Comment by koppor** [comment]
OK, I updated the description of the issue. - I really wondered why you recommended that, but who knows: sometimes old code is good :innocent:

**Comment by johannesbottcher** [comment]
So far, there are three *template* at the very top. Personally, i cannot recommend any of them.
